gcf/custom/manifest.go

```diff
@@ -193,10 +194,17 @@ func ComputeManifest(
 					manifestImport.McfProtoUrl,
 					filepath.Join("/bigstore", bucket, root, "data", im, tab, "graph.tfrecord@*.gz"),
 				)
+				manifestImport.McfUrl = append(
+					manifestImport.McfUrl,
+					filepath.Join("/bigstore", bucket, root, "data", im, tab, "graph.tfrecord@*.gz"),
```

Comment:
mcf_url should not be graph.tfrecord
mcf_url should be something like   ..../.../nodes.mcf@N

Reply:
I don't see nodes.mcf@N created in the GCS though. Will they be created if I add this?